userprofile/templates/userprofile/support.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block title %}{% blocktrans %}{{ brand_name }} Support {% endblocktrans %} | {{ block.super }}{% endblock %}
+
+{% block content %}
+
+<div class="page-header">
+	<h1>{% blocktrans %}{{ brand_name }} Support {% endblocktrans %}</h1>
+</div>
+
+<div class="row">
+	<div class="col-xs-12">
+		<h2 class="text-center">
+			{% trans "Hello" %} {% if user.first_name or user.last_name %} {{ user.first_name }} {{ user.last_name }},
```

Comment:
It would be more correct to use blocktrans for the whole introduction instead of concatenating multiple translated strings. There may be languages that use a very different order for presenting these things.

However, is it necessary to start with "Hello you"? It feels odd to me. The introduction could be done without any personal acknowledgement of the user. It is debatable which way is better. What is your argument for "hello"?

Reply:
I took some inspiration from https://support.github.com/

Reply:
We could remove the user's name and just say "hello", or skip that. Asking "how can we help you" is misleading since we can't add a search feature to this page. The page introduction should emphasize that this page lists documentation for teachers and the support contact information is there as well if they can't find solutions on their own.

The page should emphasize that this is teacher support. Students can see the page, but it is not targeted to them. Actually, the page should have a short "for students" section, but at the moment it would only say that students should contact the course staff if they have problems. The contact method varies in each course and should be described in the course pages. We could also link to the common student services for matters that belong to degree programmes or study administration, or student well-being services.
* https://into.aalto.fi/display/enopinnot
* https://into.aalto.fi/display/enopisk/Student+services